doc/cascadia/profiles.schema.json

```
@@ -339,17 +339,17 @@
         },
         "showTerminalTitleInTitlebar": {
           "default": true,
-          "description": "When set to true, titlebar displays the title of the selected tab. When set to false, titlebar displays 'Windows Terminal'.",
+          "description": "When set to true, titlebar displays the title of the selected tab. When set to false, titlebar displays \"Windows Terminal\".",
           "type": "boolean"
         },
         "snapToGridOnResize": {
           "default": false,
-          "description": "When set to true, the window will snap to the nearest character boundary on resize. When false, the window will resize 'smoothly'",
+          "description": "When set to true, the window will snap to the nearest character boundary on resize. When false, the window will resize \"smoothly\"",
```

```suggestion
          "description": "When set to true, the window will snap to the nearest character boundary on resize. When false, the window will resize smoothly",
```

just drop the quotey ones